### PR TITLE
Adding crewmate

### PR DESCRIPTION
**Pull request by lapinozz**

### CODE REVIEW

File: src/ship.hpp [hunk 1]
```diff
@@ -42,9 +42,9 @@ class Ship : public Entity
         grid.setBlockType(grid.getBlockArchetypeIdx("DoorClosed"), {2, 0});
     }
 
-    void createCrewmate(Crewmate&& crewmate)
+    Crewmate& addCrewmate(Crewmate&& crewmate)
     {
-        crewmates.emplace_back(std::move(crewmate));
+        return crewmates.emplace_back(std::move(crewmate));
```
[the remark this answers is not in the document]
Comment (lapinozz) [reply src/ship.hpp:47]: It's not meant to be stored

[the remark this answers is not in the document]
Comment (makian123) [reply src/ship.hpp:47]: Yeah but you need to guarantee you wont be adding any entities after first one

[the remark this answers is not in the document]
Comment (lapinozz) [reply src/ship.hpp:47]: No it's fine if you don't store the reference

[the remark this answers is not in the document]
Comment (makian123) [reply src/ship.hpp:47]: Code
```cpp
auto &crewmate = addCremate(...);
auto &crewmate2 = addCrewmate(...);
```
Wont work and not that well done :pensive:

**Pull request merged**